plugins/org.jkiss.dbeaver.ext.tidb.ui/plugin.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.2"?>
+<!--
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2022 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+  -->
+<plugin>
+
+    <extension point="org.jkiss.dbeaver.dataSourceProvider">
+        <editorContribution
+                editorId="org.jkiss.dbeaver.ext.mysql.ui.editors.MySQLSessionEditor"
+                category="connectionEditor" label="%editor.session_manager.name"
+                icon="platform:/plugin/org.jkiss.dbeaver.model/icons/tree/sessions.png">
+            <supports dataSource="tidb"/>
+        </editorContribution>
+    </extension>
+
+    <extension point="org.jkiss.dbeaver.dataSourceView">
+        <view
+                id="org.jkiss.dbeaver.ext.tidb.dataSourceWizardView"
+                dataSource="tidb"
+                targetID="org.jkiss.dbeaver.ext.ui.newConnectionWizard"
+                label="%dialog.connection.header"
+                class="org.jkiss.dbeaver.ext.tidb.ui.views.TiDBConnectionPage"
+                icon="platform:/plugin/org.jkiss.dbeaver.ext.tidb/icons/tidb_icon.png"/>
```

Comment:
So, you have `ext.tidb.ui/icons`, but you do not use them. Please remove them. `ext.tidb/icons` will be enough
Except for idb_logo.

Reply:
Deleted.